src/app/bridge/layout.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
Might change depending of Eugene progress

Reply:
what abou this @oktapodia ?